OrganizationChart.md

```diff
@@ -0,0 +1,196 @@
+        end
+
+        table.insert(lines, string.rep("  ", indent + 1) .. key .. " = " .. value .. ",")
+    end
+    table.insert(lines, indentStr .. "}")
+    return table.concat(lines, "\n")
+end
+
+local function mergeTablesRecursive(t1, t2)
+    for k, v in pairs(t2) do
+        if type(v) == "table" and type(t1[k]) == "table"  then
+            mergeTablesRecursive(t1[k], v)
+        else
+            t1[k] = v
+        end
+    end
+    return t1
+end
+
+local function join(tbl, sep)
+    return table.concat(tbl, sep or ", ")
+end
+local function cleanAndLower(str)
+  if str ~= nil and #str>1 then
+    local cleaned = str:gsub("[^%w]", "")
+    cleaned = str:gsub(" ", "")
+    return cleaned:lower()
+  end
+  return ""
+end
+
+local function generatePerson(uml, person, path)
+    local uml = {}
+    table.insert(uml, "Enterprise_Boundary( "..cleanAndLower(person.job.company).." , \""..person.job.company.."\") {")
+    for i,department in ipairs(person.job.department) do
+      if #department>0 then
+        table.insert(uml, string.rep(" ", i*4) .."Boundary( "..cleanAndLower(department).." , \""..department.."\") {")
+      end
+      if i== #person.job.department then
+          local cleanPerson=cleanAndLower(person.first_name.. person.last_name)
+          local pathPerson=" , \"[[/"..path.."/".. person.first_name.."%20"..person.last_name.." ".. person.first_name.." " ..person.last_name.."]]\""
+          local positionPerson=", \""..join(person.job.position, ", ").."\""
+          local spritePerson=""
+          if person.image == nil then
+            person.image="https://static.wikia.nocookie.net/villains/images/4/4a/Kevin_Spacey_John_Doe_Se7en.jpeg"
+            person.image_scale=0.1
+          end  
+          local scalePerson=0.6
+          if person.image_scale ~= nil then
+            scalePerson=scalePerson * person.image_scale
+          end
+          spritePerson=",$sprite=\"img:"..person.image.."{scale="..scalePerson.."}\""
+          table.insert(uml,  string.rep(" ", i*8).."Person( ".. cleanPerson..pathPerson..positionPerson..spritePerson..")")
+      end 
+    end
+    for i,department in ipairs(person.job.department) do
+      if #department>0 then
+        table.insert(uml, string.rep(" ",(#person.job.department-i+1)*4) .."}")
+      end
+    end
+    table.insert(uml, "}")
+    return table.concat(uml, "\n")
+end
+
+function children(path)
+    local crumbsChildren = {}
+    local mypage = path or editor.getCurrentPage()
+    for page in each(table.sort(space.listPages(), compareDate)) do
+        --print(mypage,page.name,string.find(page.name,mypage) )
+        if (string.find(page.name,mypage) and mypage ~= page.name)
+        then
+              table.insert(crumbsChildren, {name = page.ref})
+        end
+    end
+    return crumbsChildren
+end
+
+function getFrontMatter(page)
+    return index.extractFrontmatter(space.readPage(page)).frontmatter
+end 
+
+function organizationChart(path)
```

Comment:
This seems to the primary public API, perhaps it should live in `orgchart.chart` or something instead? And can you include an example of usage at the beginning of this page? There seem to be a lot of implicit assumptions in here.